frontend/javascripts/dashboard/spotlight_view.js

```diff
@@ -213,6 +213,53 @@ class SpotlightView extends React.PureComponent<PropsWithRouter, State> {
               )}
             </div>
           </Spin>
+          {features().addMissingDatasetButtonEnabled ? (
+            <div
```

Comment:
Can you extract this part in the same way as the SimpleHeader and WelcomeHeader (for example as the `MissingDatasetCard` or so)? This will help keeping the render method from growing too big and confusing.